### ISSUE TITLE
fork of xcape

### ISSUE THREAD

**Issue by Konfekt** [hanschen/ksuperkey#6]
Hello,

Thanks for this useful tool for users transitioning from Windows. Could you summarize how 

`ksuperkey`

differs from

` xcape -e 'Super_L=Alt_L|F1'` ?

I noticed that with xcape the input field of the KDE Launcher lacks often focus.

**Comment by hanschen** [comment]
It is exactly the same thing. Back in the days xcape didn't support custom bindings, which is why I forked it to create ksuperkey. Now ksuperkey is just xcape but with a different default binding. However, because ksuperkey has caught some attention and appears in guides etc., I figured I should just keep it alive.

Side note: this is also why ksuperkey doesn't appear as a fork on GitHub - I already had a different fork of xcape, and GitHub didn't allow you to create two forks of the same repository.

Hope that answers your question. I don't know why the input field of your launcher loses focus, but it shouldn't be xcape's fault.

**Comment by Konfekt** [comment]
Thank you. This explains everything.

**Comment by Konfekt** [comment]
To be clear: Since you forked the plugin, I think there have been no code but only documentation changes in xcape, so it is literally the same code, isn't it?

**Comment by hanschen** [comment]
Before the custom bindings there were some custom code, now only only line has been changed ([line 91 in xcape.c](https://github.com/hanschen/ksuperkey/blob/master/xcape.c#L91)). So the behavior should be the same as xcape.
